line_item_manager/gam_config.py

```diff
@@ -38,7 +38,8 @@ def target(key: str, names: Iterable[str], operator='IS', match_type: str='EXACT
             displayName=name,
             matchType=match_type,
         ))
-    tgt_values = TargetingValues(key_id=tgt_key['id']).fetch(create=True, recs=recs, validate=True)
+    all_values = TargetingValues(key_id=tgt_key['id']).fetch(create=True, recs=recs, validate=True)
+    tgt_values = [v_ for v_ in all_values if v_['name'] in set(names)]
```

Comment:
Why do you filter names in Python and don't use "name" property of getCustomTargetingValuesByStatement service? See https://developers.google.com/ad-manager/api/reference/v202011/CustomTargetingService#getCustomTargetingValuesByStatement  

This seems more clean:
```
tgt_values = TargetingValues(key_id=tgt_key['id'], name=list(names)).fetch(create=True, recs=recs, validate=True)
```

Reply:
I agree with you.  I updated based on your suggestion.  I did have to include an additional change in test client to get related tests to pass.